package-lock.json

```diff
@@ -22870,6 +22870,11 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=",
       "dev": true
     },
+    "typescript": {
```

Comment:
`package-lock.json` still has typescript and https to http changes, should be reset to master too.

Reply:
my bad